src/core/importType.js

```diff
@@ -72,14 +72,14 @@ function isRelativeToSibling(name) {
 }
 
 function typeTest(name, settings, path) {
-  if (isAbsolute(name, settings, path)) { return 'absolute' }
```

Comment:
these are all intentional, not bugs, to minimize future refactoring.

Reply:
Yeah, I figured they may be. I've reverted for this file. Unfortunately, LGTM's disabling only works at the individual line or file level, so disabling these warnings from LGTM would probably be considered too ugly, triggering as they would `max-len` warnings.